terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/test/java/org/terasoluna/gfw/common/codelist/i18n/SimpleI18nCodeListTest.java

```diff
@@ -116,51 +101,86 @@ public void testSetRowsByCodeList01() {
     }
 
     @Test(expected = UnsupportedOperationException.class)
-    public void testSetRowsByCodeList02() {
-        // check unmodifiable
-        testSetRowsByCodeList01.asMap(Locale.ENGLISH).put("0", "Sunday");
+    public void testAsMapCheckUnmodifiable() {
+        testSetRows.asMap(Locale.ENGLISH).put("0", "Sunday");
     }
 
     @Test
-    public void testSetColumns01() {
-        assertThat(testSetColumns01.codeListTable.size(), is(14)); // 2 rows x 7
-                                                                   // columns
+    public void testSetRows() {
+        assertThat(testSetRows.codeListTable.size(), is(14)); // 2 rows x 7
+                                                              // columns
+    }
 
-        Map<String, String> row1 = testSetColumns01.asMap(Locale.ENGLISH);
-        assertThat(row1, is(notNullValue()));
-        assertThat(row1.keySet().toString(), is("[0, 1, 2, 3, 4, 5, 6]")); // check
-                                                                           // order
-        assertThat(row1.get("0"), is("Sun."));
-        assertThat(row1.get("1"), is("Mon."));
-        assertThat(row1.get("2"), is("Tue."));
-        assertThat(row1.get("3"), is("Wed."));
-        assertThat(row1.get("4"), is("Thu."));
-        assertThat(row1.get("5"), is("Fri."));
-        assertThat(row1.get("6"), is("Sat."));
+    @Test
+    public void testSetRowsByCodeList() {
+        assertThat(testSetRowsByCodeList.codeListTable.size(), is(14)); // 2 rows x 7
+                                                                        // columns
+    }
 
-        Map<String, String> row2 = testSetColumns01.asMap(Locale.JAPANESE);
-        assertThat(row2, is(notNullValue()));
-        assertThat(row2.keySet().toString(), is("[0, 1, 2, 3, 4, 5, 6]")); // check
-                                                                           // order
-        assertThat(row2.get("0"), is("日"));
-        assertThat(row2.get("1"), is("月"));
-        assertThat(row2.get("2"), is("火"));
-        assertThat(row2.get("3"), is("水"));
-        assertThat(row2.get("4"), is("木"));
-        assertThat(row2.get("5"), is("金"));
-        assertThat(row2.get("6"), is("土"));
+    @Test
+    public void testSetColumns() {
+        assertThat(testSetColumns.codeListTable.size(), is(14)); // 2 rows x 7
+                                                                 // columns
     }
 
-    @Test(expected = UnsupportedOperationException.class)
-    public void testSetColumnss02() {
-        // check unmodifiable
-        testSetColumns01.asMap(Locale.ENGLISH).put("0", "Sunday");
+    @Test
+    public void testSetFallbackTo() {
+        testSetFallbackTo.setFallbackTo(Locale.JAPANESE);
+        testSetFallbackTo.afterPropertiesSet();
+    }
+
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
- [ ] You should assert exception message that specified on your code.

Reply:
Modified by 05779548b4928cea046e85b971cfa89c3ac978c2
Corrected it as instructed except for the following two points.

* `testAsMapCheckUnmodifiable` : Added fail method because assert target does not exist
* `testSetFallbackTo` : Add try-catch and fail method to be inverse of failed test pattern

Reply:
Why you try-catch and fail intentionally on `testSetFallbackTo`? Without try-catch, you can get expected result.

Reply:
Since the codec pointed out that "it is necessary to include one or more assert or fail in one test" in another place, we corresponded in the same way here as well.
but, I excluded try-catch as ​​it is not necessary here.
b8397315318dbc9ba2ad0727d863c3e89f91f9d1

Reply:
Adding `@Test (expected = None.class)` no longer causes warnings.
090ddf770a01304ee8140441558ab787c6309fd4